### DISCUSSION TITLE
Was Wondering about the Windows10Debloater

### DISCUSSION THREAD

**Discussion by KaptainCnuckz** [Sophia-Community/SophiApp#36] [General]
I ran the Windows10Debloater a month or so ago, before first hearing about SophiaScript/SophiApp, which triggered the in-app warning.
![Screenshot_1](https://user-images.githubusercontent.com/76413382/154803266-743f865b-6ca6-49fb-b094-65919edecfaa.jpg)

As far as stability goes, I hadn't really had issues. However, I'm well-aware the contributors are way more tech savvy than I am, so if they went out of their way to implement a warning, then it's something to be taken seriously. 

So out of curiosity I thought I'd pop in and ask; Assuming I decide to not do a fresh Win10 reinstall, what sort of issues should I be on the lookout for? i.e. The typical BSODs when system files are fudged? Or a variety of things?

**Comment by farag2** [comment]
Hi! :) There is also more stable build released that has an ability to proceed with using the app anyway, if you check the box that you continue to use it at your own risk: https://github.com/Sophia-Community/SophiApp/releases/tag/1.0.0.22 (supports Windows 11. May contain bugs).

What about the Sycnex script: just take a look at how many opened, unreplied issue it has since the release? It won't be surprise for you that that project was abandoned by him (that was stated in the Readme by him). Unfortunately, he has very poor knowledge of coding, Windows internal. Oh, don't think that we're senior C# developers. :cat:

> what sort of issues should I be on the lookout for?

What I saw on my users PCs before who had asked me for help: total instability (no BSoDs, but strange bugs), broken some features, broken system files due to Sycnex took ownership of them to remove or something else, I don't remember — just review his code. My advise: do not use just so-called debloating scripts. All that they can is to break user's OS. Unfortunately. The same cannot be said accordingly to the whole Sophia project. I hope so. :cat: We do know how to configure the OS not to break anything and we take responsibility of our actions.

**Comment by KaptainCnuckz** [comment]
I hadn't looked into it much, but saw some YouTubers praising it. Starting to wish I had after reading what you've told me. Luckily my PC's been relatively stable so far. In the future, I'll definitely be sure to read more into potential bugs or quirks with any GitHub programs that touch on any OS files. I'm often quick to trust since I know GitHub doesn't allow viruses, but I tend to forget viruses aren't the only thing to worry about when messing with important things like this.

And thank you for letting me know about the build! Ran the latest release as of typing. Things seem to have worked great. There was a few things leftover in the privacy section that you guys let me disable, that didn't manage to get caught through other means. 😄 I'll definitely be sure to use this app for any privacy edits to my OS going forward.

**Comment by farag2** [comment]
Hi). Even the latest 26 build contains 6 bugs, but we're very close to the release this or the next week.

**Comment by Carl-Gauss-root** [comment]
Hi netizens, amantes del open source y código limpio, tengo una pregunta, es la siguiente:
En conclusión: ¿ es recomendable usar el windows10Debloater, para desinflar el sistema operativo?
Gracias
